src/main/scala/devices/mockaon/WatchdogTimer.scala

```diff
@@ -60,8 +60,7 @@ class WatchdogTimer extends MultiIOModule with GenericTimer {
   lazy val io = IO(new GenericTimerIO(regWidth, ncmp, maxcmp, scaleWidth, countWidth, cmpWidth) {
```

Comment:
It's not clear to be how to successfully compile the project with new chisel3 releases. Perhaps someone could look at this? Potentially @sequencer